modules/templates/helper.go

```diff
@@ -93,6 +93,7 @@ func NewFuncMap() []template.FuncMap {
 		"TimeSinceUnix": timeutil.TimeSinceUnix,
 		"RawTimeSince":  timeutil.RawTimeSince,
 		"FileSize":      base.FileSize,
+		"PrettyNumber":  base.PrettyNumber,
```

Comment:
Any reason to go through this func when all it does is wrap the humanize func?  
Same with `FileSize` now that it uses humanize.

Reply:
Because `base.FileSize` is used all over the code and to have more flexibility just in case